controller/konnect/ops/ops_kongconsumer.go

```diff
@@ -48,10 +49,15 @@ func createConsumer(
 	}
 
 	// Set the Konnect ID in the status to keep it even if ConsumerGroup assignments fail.
-	consumer.Status.Konnect.SetKonnectID(*resp.Consumer.ID)
+	id := *resp.Consumer.ID
```

Comment:
Please add a check above this line to check if `resp`, `resp.Consumer` or `resp.Consumer.ID` is `nil`, similarly to 

https://github.com/Kong/gateway-operator/blob/c87d1ddcc75f5b5ff62ce31eff09e30c5cfc0d2a/controller/konnect/ops/ops_kongsni.go#L41-L43.

I've noticed that this can happen e.g. when we receive a network error like i/o timeout.

Reply:
Done.